tests/templates/inventory-do.j2

```diff
@@ -0,0 +1,48 @@
+{% for instance in droplets.results %}
+node{{loop.index}} ansible_ssh_host={{instance.droplet.ip_address}}
```

Comment:
FYI I changed the gce template to use the same name for ansible_hostname and inventory_hostname. Without it, upgrades with calico from v2.0.1 are broken.